CMake/cdat_modules/esmf_pkg.cmake

```diff
@@ -37,11 +37,11 @@ endif()
 if(CDAT_BUILD_ESMF_ESMP)
     if("${CMAKE_Fortran_COMPILER_ID}" STREQUAL GNU)
       # GNU gfortran must be >= 4.3
-      if(${Fortran_MAJOR_VERSION} GREATER 3 AND ${Fortran_MINOR_VERSION} GREATER 2)
+      if(${Fortran_MAJOR_VERSION} GREATER 3 AND ${Fortran_MINOR_VERSION} GREATER 1)
         ## On APPLE need to test for -arch as well!
         add_cdat_package(ESMF "" "Build ESMF" ON)
       else()
-        message(FATAL_ERROR "[ERROR] gfortran must be 4.3 <= version < 5.0; you have ${Fortran_VERSION}")
+        message(FATAL_ERROR "[ERROR] gfortran must be 4.1 <= version < 5.2; you have ${Fortran_VERSION}")
```

Comment:
ESMF was compiled with gfortran 5.2.1  
* ./lib0/Linux.gfortran.64.openmpi.default/libesmf.so
* ./lib0/Linux.gfortran.64.openmpi.default/libesmf.a

I changed 4.3 to 4.1 due to the test above, but I debated about keeping 4.3 in the ERROR message.  If somebody use gfortran 4.1 the test will pass.  I don't know if ESMF will compile though, the documentation says ESMF requires gfortran 4.3 for linux

crunchy: uses gfortran 4.4.7  and  I have gfortran 5.2.1,  that is why I used MINOR_VERSION  greater than 1.

* http://sourceforge.net/p/esmf/esmf/ci/master/tree/build_config/Linux.gfortran.default/